src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -137,7 +138,8 @@ public void Write(CompilerData compilerData, ObjectModuleDetails omDetails)
                     { "moduleLibrary", (name == library ? string.Empty : library) },
                     { "sessionId", s_sessionId },
                     { "hash", this.sha256 },
-                    { "error", string.Empty }
+                    { "error", string.Empty },
+                    { "toolVersion", VersionConstants.Version }
```

Comment:
>toolVersion [](http://example.com/codeflow?start=23&length=11)

this isn't needed.
since all lines are associated with sessionId, once we do the summarized data, everything will be "linked"

Reply:
removed